From 9540e6097563ea754b8619fe85ba010071a67244 Mon Sep 17 00:00:00 2001
From: Kamil Dudka <kdudka@redhat.com>
Date: Thu, 8 Aug 2013 12:54:53 +0200
Subject: [PATCH 2/2] runwsgi.py: start the server explicitly at 127.0.0.1

Otherwise it may start on ::1, which would consequently break self-test.
---
 tests/runwsgi.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tests/runwsgi.py b/tests/runwsgi.py
index c9bc236..096038c 100644
--- a/tests/runwsgi.py
+++ b/tests/runwsgi.py
@@ -85,7 +85,7 @@ class ServerThread(threading.Thread):
         self.app = app
         self.port = port
         self.server_kwargs = server_kwargs
-        self.server = server(host='localhost', port=self.port, **self.server_kwargs)
+        self.server = server(host='127.0.0.1', port=self.port, **self.server_kwargs)
     
     def run(self):
         bottle.run(self.app, server=self.server, quiet=True)
-- 
1.7.1

